.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+### Proposed Changes
+
+
+### Checklist
+
+- [ ] I have bumped the necessary versions
```

Comment:
Maybe update the README with information about version bumping (I'm assuming you mean chartVersion?), and link to https://semver.org